src/Textile/API/account.ts

```diff
@@ -1,88 +1,96 @@
 import { NativeModules } from 'react-native'
 import { Buffer } from 'buffer'
-import { pb } from '../Models'
 
-const { TextileNode } = NativeModules
+import {
+  IContact,
+  Contact,
+  IQueryOptions,
+  QueryOptions,
+} from './model'
+
+const { AccountBridge } = NativeModules
 
 /**
  * Get the account address.
  *
  * ```typescript
- * const address = API.account.address();
+ * const address = Textile.account.address();
  * ```
  */
 export async function address(): Promise<string> {
-  const result = await TextileNode.address()
+  const result = await AccountBridge.address()
   return result as string
 }
 
 /**
  * Get the account seed phrase to display to user.
  *
  * ```typescript
- * const seed = API.account.seed();
+ * const seed = Textile.account.seed();
  * ```
  */
 export async function seed(): Promise<string> {
-  const result = await TextileNode.seed()
+  const result = await AccountBridge.seed()
   return result as string
 }
 
 /**
  * Encrypt data with the account address.
  *
  * ```typescript
- * const encrypted = API.account.encrypt(Buffer.from(JSON.stringify({foo:"bar"})));
+ * const encrypted = Textile.account.encrypt(Buffer.from(JSON.stringify({foo:"bar"})));
  * ```
  */
 export async function encrypt(input: Buffer): Promise<Buffer> {
-  const result = await TextileNode.encrypt(input.toString('base64'))
+  const result = await AccountBridge.encrypt(input.toString('base64'))
   return Buffer.from(result, 'base64')
 }
 
 /**
  * Decrypt data previously encrypted with the account address.
  *
  * ```typescript
- * const decrypted = API.account.decrypt(encrypted);
+ * const decrypted = Textile.account.decrypt(encrypted);
  * ```
  */
 export async function decrypt(input: Buffer): Promise<Buffer> {
-  const result = await TextileNode.decrypt(input.toString('base64'))
+  const result = await AccountBridge.decrypt(input.toString('base64'))
   return Buffer.from(result, 'base64')
 }
 
 /**
  * List all own contact.
  *
  * ```typescript
- * const contact: pb.IContact = API.account.contact();
+ * const contact: IContact = Textile.account.contact();
  * ```
  */
-export async function contact(): Promise<pb.IContact | undefined> {
-  const result = await TextileNode.accountContact()
+export async function contact(): Promise<IContact | undefined> {
+  const result = await AccountBridge.contact()
```

Comment:
And the new native bridge modules use method names consistent with the native sdks and rn sdk itself (`contact` instead of `addContact`, in this case)